config/application.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I'd like to understand this a bit more:

1. How is this file even being included, if it's [explicitly gitignore'd](https://github.com/18F/identity-idp/blob/9d19a382d3009d537af6ebf96f590b43053f7981/.gitignore#L38)
2. When I checked out this branch, it blew away my local `application.yml` which is quite an inconvenience. Is that going to happen to everyone when it's merged?

Reply:
Also, when I make changes to this file locally, they're marked for commit, which I don't think we'd want.

Reply:
Should we add these comments into `application.yml.default` instead?

Reply:
How about we roll back all the `application.yml` changes, and simply update the README so that the template values can be copied and pasted in the correct place? I definitely don't want to be messing up people's workflows and this might be the best compromise

Reply:
Yeah I think that'll work 👍